src/main/java/com/cityelf/model/Address.java

```diff
@@ -0,0 +1,37 @@
+package com.cityelf.model;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "addresses")
+public class Address {
+
+  @Id
+  private long id;
+  @Column(name = "street")
```

Comment:
It'll look better if you add an extra empty line between the previous and the next property.
The same for all other entity classes below.